login/src/Components/login/LoginBody.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import img2 from "../../images/bodypic.jpg"
+import { Link } from 'react-router-dom'
+
+
+export default function LoginBody() {
+  const validateform=(e)=>{
+    e.preventDefault()
+
+    let email=document.getElementById('InputEmail').value
+    let pattern = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    if(!pattern.test(email)){
```

Comment:
please create a validation file in the util folder and keep all validations in that file.